src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "altinn-app-frontend",
-  "version": "3.27.4",
+  "version": "3.27.5",
```

```suggestion
  "version": "3.28.0",
```

New feature. We use [semver](https://semver.org/).